core/api/transport/impl/http/http_listener_impl.hpp

```diff
@@ -38,11 +41,11 @@ namespace kagome::api {
      * @param endpoint loopback ip address to listen
      * @param http_config http session configuration
      */
-    ListenerImpl(Context &context,
-                 const Configuration &configuration,
-                 HttpSession::Configuration http_config);
+    HttpListenerImpl(Context &context,
+                     const Configuration &configuration,
+                     HttpSession::Configuration http_config);
```

Comment:
HttpSession has an alias Session in this class so kinda weird to see HttpSession here.

Reply:
replaced by SessionImpl alias